diff --git a/node_modules/@capacitor/keyboard/.swiftpm/xcode/xcuserdata/ivanmuccini.xcuserdatad/xcschemes/xcschememanagement.plist b/node_modules/@capacitor/keyboard/.swiftpm/xcode/xcuserdata/ivanmuccini.xcuserdatad/xcschemes/xcschememanagement.plist
new file mode 100644
index 0000000..7b56057
--- /dev/null
+++ b/node_modules/@capacitor/keyboard/.swiftpm/xcode/xcuserdata/ivanmuccini.xcuserdatad/xcschemes/xcschememanagement.plist
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>CapacitorKeyboard.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>3</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
diff --git a/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m b/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m
index 29c0708..704b1cf 100644
--- a/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m
+++ b/node_modules/@capacitor/keyboard/ios/Sources/KeyboardPlugin/Keyboard.m
@@ -110,7 +110,7 @@ - (void)resetScrollView
 
 - (void)onKeyboardWillHide:(NSNotification *)notification
 {
-  [self setKeyboardHeight:0 delay:0.01];
+  [self setKeyboardHeight:0 delay:0.0];
   [self resetScrollView];
   hideTimer = [NSTimer scheduledTimerWithTimeInterval:0 repeats:NO block:^(NSTimer * _Nonnull timer) {
     [self.bridge triggerWindowJSEventWithEventName:@"keyboardWillHide"];
@@ -142,8 +142,7 @@ - (void)onKeyboardWillShow:(NSNotification *)notification
     }
   }
 
-  double duration = [[notification.userInfo valueForKey:UIKeyboardAnimationDurationUserInfoKey] doubleValue]+0.2;
-  [self setKeyboardHeight:height delay:duration];
+  [self setKeyboardHeight:height delay:0.0];
   [self resetScrollView];
 
   NSString * data = [NSString stringWithFormat:@"{ 'keyboardHeight': %d }", (int)height];
diff --git a/node_modules/@capacitor/keyboard/android/build.gradle b/node_modules/@capacitor/keyboard/android/build.gradle
index e69de29..d69de29 100644
--- a/node_modules/@capacitor/keyboard/android/build.gradle
+++ b/node_modules/@capacitor/keyboard/android/build.gradle
@@ -42,7 +42,7 @@
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
