diff --git a/node_modules/@capacitor-community/sqlite/Package.swift b/node_modules/@capacitor-community/sqlite/Package.swift
new file mode 100644
index 0000000..d2ff9f5
--- /dev/null
+++ b/node_modules/@capacitor-community/sqlite/Package.swift
@@ -0,0 +1,24 @@
+// swift-tools-version: 5.9
+import PackageDescription
+
+let package = Package(
+    name: "CapacitorCommunitySqlite",
+    platforms: [.iOS(.v13)],
+    products: [
+        .library(
+            name: "CapacitorCommunitySqlite",
+            targets: ["CapacitorCommunitySqlite"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "6.0.0")
+    ],
+    targets: [
+        .target(
+            name: "CapacitorCommunitySqlite",
+            dependencies: [
+                .product(name: "Capacitor", package: "capacitor-swift-pm"),
+                .product(name: "Cordova", package: "capacitor-swift-pm")
+            ],
+            path: "ios/Plugin")
+    ]
+)
diff --git a/node_modules/@capacitor-community/sqlite/android/.project b/node_modules/@capacitor-community/sqlite/android/.project
new file mode 100644
index 0000000..d42629f
--- /dev/null
+++ b/node_modules/@capacitor-community/sqlite/android/.project
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>capacitor-community-sqlite</name>
+	<comment>Project capacitor-community-sqlite created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1770134898501</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
